assembl/tests/views_tests/test_graphql_user.py

```diff
@@ -42,6 +42,33 @@
 }
 """
 
+UPDATE_COOKIES_INFORMATION_FOR_USER_MUTATION = u"""
+mutation updateAcceptedCookies($actions: [CookieTypes]!) {
+    updateAcceptedCookies(
+    actions: $actions
+    ) {
+    user{
+    ... on AgentProfile {
+      id
+      acceptedCookies
+      }
+    }
+  }
+}
+"""
+
+QUERY_COOKIES_INFORMATION_FOR_USER = u"""
+query User($id: ID!) {
+  user: node(id: $id) {
+    ... on AgentProfile {
+      id
+      name
+      acceptedCookies
+    }
+  }
+}
+"""
+
```

Comment:
You should really use graphql_registry in the tests so you are sure you are testing the same mutations that is used in reality.

Reply:
:+1: